go/fn/run.go

```diff
@@ -16,6 +16,10 @@ func AsMain(p ResourceListProcessor) error {
 	}
 	out, err := Run(p, in)
 	if err != nil {
+		_, outErr := os.Stdout.Write(out)
```

Comment:
https://github.com/kubernetes-sigs/kustomize/pull/4319/files#diff-68b7beaf7e26d0e42b157805ce66c691d04eeb739615053e390c17c80c625fb4R174-R199
This is how I handle it in the upstream PR. It may be useful to look at.

Reply:
Thanks! I used it as a reference and updated it here accordingly.